### DISCUSSION TITLE
My 1 wallet has disappeared

### DISCUSSION THREAD

**Discussion by staump** [polymorpher/one-wallet#217] [General]
My 1wallet as disappeared. My wallets are no longer showing up. The wallet with ONE in it is completely gone. How do I recover this?

**Comment by polymorpher** [comment]
As I said in https://github.com/polymorpher/one-wallet/issues/214, please at least provide your wallet address, so people have somewhere to start helping you on this. Some other helpful information include, for example, whether you cleared browser storage, what browsers you used, what did you do with the wallet. With no information provided from you, there is nothing we can do.

**Comment by staump** [comment]
Im sorry .. Im not familiar with this system and did not see any response just that my post was deleted so I posted again. Im not sure the wallet address but I am using chrome and have not cleared browser storage recently. Wallet was up just 2 nights ago. Any suggestions on locating my wallet address? I thought all i needed was to use google authenticator.

**Comment by polymorpher** [comment]
Sorry for the confusion. Address is still needed in this version. Who sent ONEs to that wallet? Any transaction to trace?

**Comment by staump** [comment]
I can try to find the address. I really thought only Google Authenticator was needed; I have a few main wallets and a good amount of transactions on each. Any thing I can look for to make the digging easier?

**Comment by staump** [comment]
OK... Found it!!... Im so sorry for my lack of understanding this entire process . Im still learning. Hope this helps in some more. I am happy to contribute any information that might help someone in the future who's wallet disappears.

**Comment by polymorpher** [comment]
Can you post the transaction here?

**Reply by Mike3-www** [reply]
Same issue -wallet is gone and dont know how to recover -
one1draf2a8n4t3ckksywq87gvf5pc6dmtextjkzvq

**Reply by polymorpher** [reply]
What was the issue specifically?

**Comment by polymorpher** [comment]
I am moving this to discussions, since it is a non-technical support issue
